themes/default/content/docs/pulumi-cloud/webhooks.md

```diff
@@ -185,9 +187,15 @@ follow the link below to quickly get started with a pre-defined Slack app manife
 
 <div class="btn btn-secondary"><a target="_blank" href="https://api.slack.com/apps?new_app=1&manifest_yaml=display_information%3A%0A%20%20name%3A%20pulumi-slack-notifications%0A%20%20description%3A%20Funnel%20Pulumi%20webhooks%20to%20Slack%0A%20%20background_color%3A%20%22%238a3391%22%0Afeatures%3A%0A%20%20bot_user%3A%0A%20%20%20%20display_name%3A%20pulumi-slack-notifications%0A%20%20%20%20always_online%3A%20false%0Aoauth_config%3A%0A%20%20scopes%3A%0A%20%20%20%20bot%3A%0A%20%20%20%20%20%20-%20incoming-webhook%0Asettings%3A%0A%20%20org_deploy_enabled%3A%20false%0A%20%20socket_mode_enabled%3A%20false%0A%20%20token_rotation_enabled%3A%20false" class="tile h-full">Create a Slack app from manifest</a></div>
 
+### Microsoft Teams Webhooks
+
+Webhooks formatted for Microsoft Teams allow you to seamlessly integrate notifications about your Pulumi stacks and organizations
+into your Microsoft Teams workspace by simply providing a [Microsoft Teams incoming webhook URL](https://docs.microsoft.com/en-us/microsoftteams/platform/webhooks-and-connectors/how-to/add-incoming-webhook)
+and optionally choosing which events you want delivered using [event filters](#event-filtering).
+
 ### Deployment Webhooks
```

Comment:
Are these specific to Pulumi Deployments? If so, I'd probably change this at least to Deployments, if not Pulumi Deployments, to better distinguish them from regular ol' deployment notifications):

```suggestion
### Pulumi Deployments Webhooks
```

I can't see what these look like in the UI, so this is assuming they show up as Pulumi Deployments as a destination. If not feel free to adjust accordingly!

Reply:
The UI calls it "Deployment" so will leave this as-is.